tests/__tests__/sitka_module.test.ts

```diff
@@ -67,21 +67,61 @@ describe("SitkaModule", () => {
   })
 
   test('able to get defaultState', () => {
+    const expected = { 
+      size: 12, 
+      value: 'Hello World', 
+      numberOfEdits: 0, 
+      history: [] 
+    }
+    const actual = textModule.defaultState
+    expect(actual).toEqual(expected)
   })
 
   test('able to get moduleName', () => {
+    const actual = textModule.moduleName
+    expect(actual).toEqual("text")
   })
 
   test('able to get modules', () => {
+    const expected = ["color", "text"]
+    const actual = Object.getOwnPropertyNames(textModule.modules)
+    expect(actual).toEqual(expected)
   })
 
   test('reduxKey returns key', () => {
+    const actual = textModule.reduxKey()
+    expect(actual).toEqual("text")
   })
 
   test('setState (protected) updates redux state with handleText (public)', () => {
+    const newState = {
```

Comment:
we can do the same here... create an `newTextModuleState` object that you can reuse.